packages/blade/src/components/Dropdown/types.ts

```diff
@@ -14,27 +14,6 @@ type DropdownProps = {
    */
   onOpenChange?: (isOpen: boolean) => void;
   selectionType?: 'single' | 'multiple';
-  /**
-   * ## Deprecated ⚠️
```

Comment:
Breaking change. We should mention it in the release notes. Should we even do this breaking change now? Too many breaking changes happening with v11 anyway

Reply:
For this also we have codemod.

I don't see harm in removing it since it was deprecated for a long time and Dropdown isn't as heavily used as Text or Input so might just break at few places. We should mention it in manual migration for sure. 

I have added both surfaceLevel and onDimiss changes to blade migration doc, Nitin is adding all changes from that doc to codemod and migration guide.